### ISSUE TITLE
Window title - Live updating

### ISSUE THREAD

**Issue by dgrumieaux** [tsgrp/hpi#756]
This enhancement would be nice to show if a user is in "Search" or "Stage" or something to that effect.

Right now we have a static title - we can set it in overlay but it will not change once the application is loaded. What about:

- Look at trello (up top) - See how it changes when we navigate to different cards?
- Easy to set - ```window.title = ""```
- Could be event driven manually, or automatically by the router?

@gsteimer - thoughts about priority or how this can/should be used? Migrated this issue from the Trello board.

**Comment by gsteimer** [comment]
Let's try to get this in for 2.4.  Do you think a new hire could do this?

**Comment by dgrumieaux** [comment]
Yep, I think we should get a little design in for this. Identifying the cases where we want this to change, and then ensuring that it behaves appropriately when 1) someone navigates to the page via refresh or initial page load, and 2) when someone navigates to the page through the regular backbone router.

@zkeller - I think we'd want to do this enhancement in our routers. Do you agree?

**Comment by zkeller** [comment]
Yeah, I think for the very basic cases, this should be easy to implement through the routers. If we wanted to get fancy (show 'View Properties' when a user clicks that action), we would obviously need to update code elsewhere, but if we just want 'Stage', 'Dashboard', 'Collections', etc, that shouldn't be difficult.

**Comment by dgrumieaux** [comment]
Then I propose that we make this enhancement about the simpler cases and we can open a "window title microstates" issue if we want to support that level of detail.
